### PR TITLE
Issue/347 dailyidea email

### PR DESCRIPTION
**Pull request by vedmant**
#347

### CODE REVIEW

File: back/mail-processing/package.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
     "sls:deploy-dev": "env-cmd -f ./conf/dev.env sls deploy",
     "sls:deploy-prod": "env-cmd -f ./conf/prod.env sls deploy",
     "sls:deploy-ci": "serverless deploy",
-    "run-dev": "env-cmd -f ./conf/dev.env ./run.py"
+    "run-dev": "env-cmd -f ./conf/dev.env ./run.py",
+    "compile-emails": "node mail-mjml/compile-emails.js"
```
Comment (aldarund) [inline back/mail-processing/package.json:11]: should compile emails command be executed before deploy on CI ?

Comment (vedmant) [reply back/mail-processing/package.json:11]: Yes, they can be executed before deploy automatically

Comment (aldarund) [reply back/mail-processing/package.json:11]: can or should ? :)

Comment (vedmant) [reply back/mail-processing/package.json:11]: It depends on the point of view...

**Pull request merged**